lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -308,7 +327,7 @@ const Reportback = {
         'isListItem': true,  // @TODO: probably a better way to address this... :insert shruggy emoji here:
         'reactions': {
           enabled: setting('dsKudosReactions.enabled'),
-          data: this.getReactions(items[i].kudos.data),
+          data: setting('dsKudosReactions.likedByCurrentUser') ? this.getReactions(items[i].kudos, true) : this.getReactions(items[i].kudos.data, false),
         }
```

Comment:
@DFurnes I was thinking that but then I _think_ we'd run into trouble [here](https://github.com/DoSomething/phoenix/blob/dev/lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js#L189), since we don't have the Kudos object?